layouts/partials/contact-identifier.html

```diff
@@ -10,8 +10,7 @@
       <div class="grid-row">
         <div class="tablet:grid-col-auto">
           <div class="vote-txt">Vote.gov</div>
-          <img class="vote-logo" loading="lazy" width="130" height="160" src="{{ .Site.BaseURL }}/assets/img/ballot_box-blue.svg"
-            alt="{{ $translation.footer.vote_logo_alt_text }}" aria-label="Vote.gov">
+          <img class="vote-logo" loading="lazy" width="130" height="160" src="{{ .Site.BaseURL }}/assets/img/ballot_box-blue.svg" role='presentation'>
```

Comment:
We need to have an empty alt attribute here `alt=""` or else it will get flagged in an accessibility scan. Test this using the WAVE chrome extension.